Formula/git.rb

```diff
@@ -28,7 +29,7 @@ class Git < Formula
   uses_from_macos "zlib", since: :high_sierra
 
   on_linux do
-    depends_on "linux-headers@4.4"
+    depends_on "linux-headers@5.15"
```

Comment:
Why is this needed? And, in particular, why are the headers still needed at runtime?

Reply:
The runtime dependency is weird - it could just be an oversight.  I will do a local test to see why it’s needed.